csrankings.csv

```diff
@@ -7553,9 +7553,8 @@ Maarten van Steen,VU Amsterdam,http://www.distributed-systems.net,1mFRcc0AAAAJ
 Maciej Piróg,University of Wroclaw,https://www.cs.ox.ac.uk/people/maciej.pirog,xxwcLlAAAAAJ
 Madalina Fiterau,University of Massachusetts Amherst,http://www.cs.cmu.edu/~mfiterau,NTHsaUQAAAAJ
 Madeleine Udell,Cornell University,https://people.orie.cornell.edu/mru8,tZ9pEDMAAAAJ
-Madhav Marathe,Virginia Tech,https://www.bi.vt.edu/faculty/Madhav-Marathe,cLjMQqsAAAAJ

```

Comment:
You *must* change both aliases.

Reply:
Just realized that other two people's entry need to be fixed as well, and it's done.